modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -414,10 +452,24 @@ private void _deleteConfiguration() throws Exception {
 		}
 	}
 
-	private void _testFactoryConfiguration(char separator) throws Exception {
+	private Configuration _testFactoryConfiguration(char separator)
+		throws Exception {
+
 		String factoryConfigurationName = StringBundler.concat(
 			StringUtil.randomId(), CharPool.DASH, StringUtil.randomId());
 
+		String testValue = StringUtil.randomId();
+
+		_testFactoryConfiguration(
+			separator, factoryConfigurationName, testValue);
+
+		return _configuration;
+	}
```

Comment:
@kyle-miho, for the kind of method _test*(), please don't return any object and keep the original `void`. Like other methods `private Configuration _createConfiguration()`, we may return Object when using the kind of methods.